react-frontend/components/App.tsx

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/*
+ * Copyright 2019-2021 Bloomreach
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BrComponent, BrPage, BrPageContext } from '@bloomreach/react-sdk';
+import { Configuration, PageModel } from '@bloomreach/spa-sdk';
+import axios from 'axios';
+import { Container, Navbar, Image, Row, Col } from 'react-bootstrap';
+import { getCookieConsentValue } from 'react-cookie-consent';
+import { useMemo, useState } from 'react';
+import { CommerceApiClientFactory, CommerceConnectorProvider } from '@bloomreach/connector-components-react';
+import { Cookies, CookiesProvider } from 'react-cookie';
+import {
+  BannerCollection,
+  BannerCTA,
+  BrCookieConsent,
+  BrPixel,
+  CategoryHighlight,
+  Content,
+  ContentPage,
+  Link,
+  Images,
+  Map,
+  Menu,
+  MultiBannerCarousel,
+  Navigation,
+  PageCatalog,
+  PathwaysRecommendations,
+  Product,
+  ProductGrid,
+  ProductHighlight,
+  SearchBar,
+  SingleBannerCarousel,
+  SingleBannerCarouselX,
+  TitleAndText,
+  Video,
+} from '.';
+import { CommerceContextProvider } from './CommerceContext';
+
+import styles from './App.module.scss';
+import { Meta } from './Meta';
+import { CommerceConfig } from '../src/utils';
+
+interface AppProps {
+  configuration: Omit<Configuration, 'httpClient'>;
+  page: PageModel;
+  commerceConfig: CommerceConfig;
+  commerceClientFactory?: CommerceApiClientFactory;
+  apolloState?: string;
+  cookies?: Record<string, string>;
+}
+
+export function App({
+  configuration,
+  page,
+  commerceConfig,
+  commerceClientFactory,
+  apolloState,
+  cookies,
+}: AppProps): JSX.Element {
+  const [, setCookieConsentVal] = useState<boolean>();
+  const mapping = {
```

Comment:
Deze moet nog opgeschoond worden zodat hier alleen de mapping in staat die we gebruiken

Reply:
doen we later wel.